lib/membrane_file/source.ex

```diff
@@ -17,18 +17,24 @@ defmodule Membrane.File.Source do
                 spec: pos_integer(),
                 default: 2048,
                 description: "Size of chunks being read"
+              ],
+              content_format: [
+                spec: module() | nil,
+                default: nil,
+                description: "Module for `RemoteStream.t()` caps sent by output pad"
               ]
```

Comment:
Still don't think this is the way. Whatever accepts remote stream should accept `nil` as the content format.